bundle/manifests/gitops-operator.clusterserviceversion.yaml

```diff
@@ -48,6 +48,7 @@ metadata:
               }
             },
             "dex": {
+              "openShiftOAuth": true,
```

Comment:
modification to csv should be done by kustomize and re-generate csv/bundle.

Reply:
I will take a look. Thanks 🙏

Reply:
Added the required changes to the samples and got them added to csv by running `make bundle`